<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlWidget</class>
 <widget class="QWidget" name="ControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ControlWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton{
 	font-family:Microsoft Yahei;
    font-size:11pt;
	color: rgb(0, 0, 0);
}
.QPushButton:hover,.QToolButton:hover{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #F2F9FF,stop:1 #DAEFFF);
}
.QPushButton:pressed,.QToolButton:pressed{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #DEF0FE,stop:1 #C0DEF6);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>45</number>
     </property>
     <item>
      <widget class="QToolButton" name="list_toolbutton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>列表显示</string>
       </property>
       <property name="icon">
        <iconset resource="EControlSys.qrc">
         <normaloff>:/EControlSys/res/list.png</normaloff>:/EControlSys/res/list.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="replay_toolbutton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>操作复盘</string>
       </property>
       <property name="icon">
        <iconset resource="EControlSys.qrc">
         <normaloff>:/EControlSys/res/replay.png</normaloff>:/EControlSys/res/replay.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="list_widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>任务名称：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="placeholderText">
             <string>请输入</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>人员：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="placeholderText">
             <string>请输入</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>重置</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <property name="styleSheet">
           <string notr="true">border-left:none;
border-top:none;
border-bottom:none;</string>
          </property>
          <property name="autoScrollMargin">
           <number>30</number>
          </property>
          <property name="indentation">
           <number>30</number>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="headerMinimumSectionSize">
           <number>25</number>
          </attribute>
          <attribute name="headerDefaultSectionSize">
           <number>50</number>
          </attribute>
          <column>
           <property name="text">
            <string>11</string>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
          </column>
          <item>
           <property name="text">
            <string>全部</string>
           </property>
           <item>
            <property name="text">
             <string>导弹测试</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>惯导测试/标定</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>引信测试</string>
            </property>
           </item>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTableWidget" name="tableWidget">
            <column>
             <property name="text">
              <string>序号</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>任务编号</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>任务名称</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>科目</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>任务模式</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>任务分类</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>人员</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>预计训练时间</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>实际开始时间</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>实际结束时间</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>状态</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>操作</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="styleSheet">
             <string notr="true">QFrame{
border:none
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">image: url(:/EControlSys/res/left.png);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9"/>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">image: url(:/EControlSys/res/right.png);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_2">
               <item>
                <property name="text">
                 <string>5条/页</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>10条/页</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>15条/页</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>20条/页</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>跳至</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_3">
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>页</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="replay_widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>科目名称：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>完成时间：</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit">
          <property name="accelerated">
           <bool>false</bool>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="displayFormat">
           <string>yyyy-MM-dd</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>重置</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget_2">
        <column>
         <property name="text">
          <string>序号</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>所属任务编号</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>所属任务名称</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>科目</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>动作名称</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>人员</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>完成时间</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="acStart">
   <property name="text">
    <string>开始</string>
   </property>
   <property name="toolTip">
    <string>开始</string>
   </property>
  </action>
  <action name="acPause">
   <property name="text">
    <string>暂停</string>
   </property>
  </action>
  <action name="acStop">
   <property name="text">
    <string>终止</string>
   </property>
   <property name="toolTip">
    <string>终止</string>
   </property>
  </action>
  <action name="acDetail">
   <property name="text">
    <string>查看详情</string>
   </property>
  </action>
  <action name="acDelTask">
   <property name="text">
    <string>删除任务</string>
   </property>
  </action>
  <action name="acEditTask">
   <property name="text">
    <string>编辑任务</string>
   </property>
  </action>
  <action name="acReplay">
   <property name="text">
    <string>复盘回放</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="EControlSys.qrc"/>
 </resources>
 <connections/>
</ui>
